library/asn1parse.c

```diff
@@ -139,17 +139,18 @@ int mbedtls_asn1_get_bool( unsigned char **p,
     return( 0 );
 }
 
-int mbedtls_asn1_get_int( unsigned char **p,
-                  const unsigned char *end,
-                  int *val )
+static int asn1_get_tagged_int( unsigned char **p,
+                        const unsigned char *end,
```

Comment:
Style: please align the parameters, i.e.
```
static int asn1_get_tagged_int( unsigned char **p,
                                const unsigned char *end,
                                int tag, int *val )
```
We generally don't fix alignment errors after the fact because it's noise in the history, but since you're modifying that line anyway, please fix the indentation.

Reply:
Done.